index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Code Your Future | CYF</title>
+    <link rel="stylesheet" href="style.css">
+   
+  
+</head>
+
+
+
+<body>
+    
+    <div class="container1">
+        <h1 class="website-title">CYF</h1>
+            <nav>
+                <ul class="links">
+                    <li><a href="#">Menu 1</a></li>
+                    <li><a href="#">Menu 2</a></li>
+                    <li><a href="#">Menu 3</a></li>
+                </ul>
+            </nav>
+    </div>
+
+    <!-- Articles-->
+
+    <div id="main" class="articles">
+        <div class="title"> About CYF</div>
+        <!--First Article-->
+        <article class="article1">
+            <div class="article-title ">
+                What We Do
+            </div>
+            <p class="article1-body">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+                standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it
+                to make a type specimetially unchanged. It was popularised in the 1960s with the release of Letraset
+                sheets containing  Lorem Ipsum passages, and more recently with desktop publishing software like Aldus
+                PageMaker including versions of Lorem IpsumIpsumIt has survived not only five centuries, but also the leap into electronic
+                typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset
+                sheets containing  Lorem Ipsum passages, and more recently with desktop publishing software like Aldus
+                PageMaker including versions of Lorem Ipsum. </p>
+        </article>
+
+        <!--Second Article-->
+        <article class="article2">
+            <div class="article-title "> Our focus </div>
+            <ul class="article2-list">
+                <li class="article-body-item ">
+                    Interactive
+                </li>
+                <li class="article-body-item">
+                    Mobile
+                </li>
+                <li class="article-body-item">
+                    Technology
+                </li>
+            </ul>
+            <p class="download">
+                <a href="#">Click here to download a pdf for our recent work </a>
+            </p>
+        </article>
+
+        <!--News-->
+        <div class="news">
+            <div class="title ">
+                News
+            </div>
+            
+                
+                <p><a href="#" id="show">Show/Hide news items</a></p>
+                <div class="content" id="mybutton">
```

Comment:
also in general don't use things like "my" in variable names - this is bad practice